commons/src/main/java/com/powsybl/commons/reporter/ReporterModelDeserializer.java

```diff
@@ -41,29 +46,57 @@ public ReporterModel deserialize(JsonParser p, DeserializationContext ctx) throw
         Map<String, String> dictionary = Collections.emptyMap();
         JsonNode dicsNode = root.get("dics");
         if (dicsNode != null) {
-            JsonNode dicNode = dicsNode.get(getDicName(ctx));
+            String dictionaryName = getDictionaryName(ctx);
+            JsonNode dicNode = dicsNode.get(dictionaryName);
+            if (dicNode == null && dicsNode.fields().next() != null) {
+                Map.Entry<String, JsonNode> firstDictionary = dicsNode.fields().next();
+                dicNode = firstDictionary.getValue();
+                LOGGER.warn("Cannot find `{}` dictionary, taking first entry (`{}`)", dictionaryName, firstDictionary.getKey());
+            }
             if (dicNode != null) {
                 dictionary = codec.readValue(dicNode.traverse(), new TypeReference<HashMap<String, String>>() {
                 });
+            } else {
+                LOGGER.warn("No dictionary found! `dics` root entry is empty");
             }
+        } else {
+            LOGGER.warn("No dictionary found! `dics` root entry is missing");
         }
         return ReporterModel.parseJsonNode(root.get("reportTree"), dictionary, codec);
     }
 
-    private String getDicName(DeserializationContext ctx) throws JsonMappingException {
-        Object dicNameInjected = ctx.findInjectableValue(DICTIONARY_VALUE_ID, null, null);
-        return dicNameInjected instanceof String ? (String) dicNameInjected : "default";
+    private String getDictionaryName(DeserializationContext ctx) {
+        try {
+            Object dicNameInjected = ctx.findInjectableValue(DICTIONARY_VALUE_ID, null, null);
+            return dicNameInjected instanceof String ? (String) dicNameInjected : DICTIONARY_DEFAULT_NAME;
+        } catch (JsonMappingException e) {
```

Comment:
Just a question: what could cause `JsonMappingException`? Is it something worthy to put a warning?

Reply:
The exception is raised if the injectable value does not exist. So, yes you're right, it's worth a warning! or at least a log with info level